docs/community_resources/faq.md

```diff
@@ -0,0 +1,90 @@
+
+If you want to read more about languages managing in eZ Platform please see the following doc pages:
+
+- https://doc.ezplatform.com/en/latest/guide/internationalization/#back-office-languages
+- https://doc.ezplatform.com/en/latest/cookbook/setting_up_multi_language_siteaccesses/
+
+#### How should I proceed with contributions to eZ Platform?
+
+First, create an issue in our [issue tracker](https://jira.ez.no/browse/EZP) and refer to it in commits and pull request headers. For example: `EZP-20104: ContentController should return error status when content is not found`. Then, you can fork our repository and create your own branch from the version of the package which you want to contribute to. 
+
+Now you can introduce whatever changes you wish, either modifying existing files or creating new ones. Once you are done with your edits, add your files to the staging area, then commit with a short, clear description and push changes to your fork. Finally, you can go to the project's page on GitHub and you should see a `Compare and pull request` button. Activate it, write a description and select `Create pull request`. Start the pull request's name with the issue number. Now you can wait for your changes to be reviewed and eventually tested and merged into the product.
+
+Here you can find more details regarding contribution procedure (provided resources apply to different repositories, but the whole process is more or less the same):
+- https://doc.ezplatform.com/en/latest/community_resources/documentation/#contributing-through-git
+- https://github.com/ezsystems/ezpublish-kernel#contributing
+
+!!! tip
+    
+    To deal with patches for packages which are not yet released, you can use [composer-patches](https://github.com/cweagans/composer-patches).
+
+#### How to clear the cache properly?
+
+Clearing cache is also covered by our [documentation](../guide/devops/#cache-clearing), it applies to file and content (http/persistence) cache.
+
+Useful commands:
+
+- clearing Symfony cache
+```bash
+php bin/console cache:clear --env prod
+```
+- clearing Redis cache
+```bash
+php bin/console cache:pool:clear cache.redis
+```
+- clearing Memcached cache
+```bash
+php bin/console cache:pool:clear cache.memcached
+```
+- clearing the Symfony cache manually
+```bash
+rm -rf var/cache/*
+```
+
+!!! caution "Clearing cache manually"
+
+    It should be executed with caution. `rm -rf var/cache/*` wipes all the files and unlike `cache:clear` doesn't warm up the cache. It reflects in significant performance drop on first request thus shouldn't be called on production environment. Besides, it could lead to issues with files ownership after running `cache:clear` as a root.
+    
+#### Where should I place my configuration files?
+
+In order to avoid overriding important configuration settings during upgrade we should separate them from eZ maintained YAML files. There are two basic approaches to achieve that goal:
```

Comment:
```suggestion
In order to avoid "merge conflicts" on important configuration settings during upgrades, moving as much as possible of your configuration to own files can be a good idea. There are two basic approaches to achieve that goal:
```